packages/artifacts/prepare-abi.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+
+# Exit on error
+set -e
+
+# Constants
+CONTRACTS_FOLDER="../contracts"
+TARGET_ABI_FILE="./src/abi.ts"
+
+# Move into contracts package and install dependencies
+cd $CONTRACTS_FOLDER
+
+yarn install && yarn build
+
+# Move back to artifacts package
+cd - > /dev/null
+
+# Wipe the destination file
+echo > $TARGET_ABI_FILE
```

Comment:
A change I did in all other packages:

```suggestion
echo "// NOTE: Do not edit this file. It is generated automatically and it will be eventually wiped." > $TARGET_ABI_FILE
```